core/commands/storage/upload/upload/do_guard.go

```diff
@@ -98,6 +102,29 @@ func doGuard(rss *sessions.RenterSession, res *escrowpb.SignedPayinResult, fileS
 	if err != nil {
 		return fmt.Errorf("failed to send challenge questions to guard: [%v]", err)
 	}
+	go func() {
+		for {
+			after := time.After(10 * time.Second)
+			select {
+			case <-rss.Ctx.Done():
+				return
+			case <-after:
```

Comment:
this is never working .. because if it doesn't hit 10s, will run default, if it hits 10s, it will continue and hit default again